scipy/signal/tests/test_signaltools.py

```diff
@@ -758,8 +758,7 @@ def test_many_sizes(self, n):
 
         out = fftconvolve(a, b, 'full', axes=[0])
         assert_allclose(out, expected, atol=1e-10)
-
-
+    
```

Comment:
Removing this newline is a PEP8 regression (want two newlines before function def), please revert. Also it's not clear why it shows two newlines deleted and one added, is there an invisible character on this line or something?

Reply:
I'm not quite sure what happened, I think my flake8 setup did something funky here.

Reply:
Newlines are still being removed in a few places.